src/HatchwaysBlog.test.js

```diff
@@ -1,8 +1,114 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { render, screen, within } from "@testing-library/react";
```

Comment:
we should add the "Do not modify this file" type of thing here

Reply:
Good call! Added a warning message.